paper-server/src/main/java/io/papermc/paper/registry/data/PaperWolfVariantRegistryEntry.java

```diff
@@ -75,6 +83,12 @@ public Builder tameClientTextureAsset(final ClientTextureAsset tameClientTexture
             return this;
         }
 
+        @Override
+        public Builder spawnConditions(final List<SpawnConditionPriority> spawnConditions) {
+            this.spawnConditions = PaperSpawnConditions.fromApi(spawnConditions, this.conversions);
```

Comment:
need to do the null check with asArgument like the other calls, seems the damage type entry misses a lot of those too. Also it's useless to check if the value is configured in the build() method since it's never null